src/main_app/jobs_workers/public_jobs_workers/copy_svg_langs/steps/svg_translate_utils/adds_utils.py

```
@@ -0,0 +1,229 @@
+
+        new_data = self._add_from_titles(titles_new, new_keys)
+        if new_data:
+            new_translations.update(new_data)
+            self.changes = True
+        else:
+            self.changes = False
+
+
+def add_translations_from_titles(translations: dict[str, Any] | ExtractorData) -> dict[str, Any]:
+    """Insert new translations into the translations dictionary."""
+    object = ExtractorData.from_any(translations)
```

Ruff (0.16.1)

[error] Variable `object` is shadowing a Python builtin

(A001)